app/components/meeting-form.js

```diff
@@ -29,41 +33,50 @@ export default class MeetingForm extends Component {
   }
 
   @task
-    *loadData() {
-      if (this.zitting.get("id")) {
-        this.bestuursorgaan = yield this.zitting.get("bestuursorgaan");
-        this.secretaris = yield this.zitting.get("secretaris");
-        this.voorzitter = yield this.zitting.get("voorzitter");
-        if (this.bestuursorgaan) {
-          yield this.fetchPossibleParticipants.perform();
-        }
-        yield this.fetchParticipants.perform();
-        yield this.fetchTreatments.perform();
+  *loadData() {
+    if (this.zitting.get("id")) {
+      this.bestuursorgaan = yield this.zitting.get("bestuursorgaan");
+      this.secretaris = yield this.zitting.get("secretaris");
+      this.voorzitter = yield this.zitting.get("voorzitter");
+      if (this.bestuursorgaan) {
+        yield this.fetchPossibleParticipants.perform();
       }
+      yield this.fetchParticipants.perform();
+      yield this.fetchTreatments.perform();
+    }
   }
 
   @task
   *fetchParticipants() {
     let queryParams = {
-      include: 'is-bestuurlijke-alias-van',
+      include: 'is-bestuurlijke-alias-van,status',
       sort: 'is-bestuurlijke-alias-van.achternaam',
       'filter[aanwezig-bij-zitting][:id:]': this.zitting.get('id'),
       page: { size: 100 } //arbitrary number, later we will make sure there is previous last. (also like this in the plugin)
     };
     const mandatees = yield this.store.query('mandataris', queryParams);
-    this.aanwezigenBijStart = Array.from(mandatees.filter( (mandatee) => isEmpty(mandatee.einde) || mandatee.einde > new Date()));
+    this.aanwezigenBijStart = Array.from(
+      // TODO: should this not filter on meeting date?
+      mandatees.filter( (mandatee) => (isEmpty(mandatee.einde) || mandatee.einde > new Date()) && [statusEffectief, statusWaarnemend].includes(mandatee.get("status.id")))
```

Comment:
Maybe the filter is too complex, I would put it in different lines so we can understand it better

Reply:
Also the TODO is resolved?